src/svg-pan-zoom.js

```diff
@@ -30,6 +30,10 @@ var optionsDefaults = {
 , customEventsHandler: null
 , eventsListenerElement: null
 , onUpdatedCTM: null
+, triggerUpOnLeave: true // This allows content to be panned even if the panning pointer is on top of a menu or outside browser
```

Comment:
Instead of having to explain it in a comment, it's better to give the prop as descriptive name as possible. 
For example here we can use `stopPanningOnLeave`